aspnetcore/mvc/views/tag-helpers/built-in/AnchorTagHelper.md

```diff
@@ -34,35 +32,110 @@ The speaker controller used in attribute definitions below is shown here.
 
 ### asp-controller
 
-`asp-controller` is used to associate which controller will be used to generate the final URL. The only valid choices are controllers that exist in the current project. To get to a list of all speakers specifying `asp-controller="Speaker"` is required. If only the `asp-controller` and no `asp-action` is specified, the default asp-action will be the name of the the controller method calling this view page.
+`asp-controller` is used to associate which controller will be used to generate the URL. The controllers specified must exist in the current project. The following code lists all speakers: 
+
+```
+<a asp-controller="Speaker" asp-action="Index" >All Speakers</a>
+```
+
+The generated URL will be
+
+```
+<a href="/Speaker">All Speakers</a>
+```
+
+If the `asp-controller` is specified and `asp-action` is not, the default `asp-action` will be the default controller method of the currently executing view.  That is, in the above example, if `asp-action` is left out, and this Anchor Tag Helper generated from the Home index view (**/Home**), the generated markup will be:
+
+
+```html
+<a href="/Home">All Speakers</a>
+```
 
 - - -
   
 ### asp-action
 
-`asp-action` is the name of the method in the controller that will be included in the final URL. That is, in the example, if the route to the Speaker Detail page is wanted, then the attribute should be set to `asp-action=Detail`. You should always set `asp-controller` when specifying `asp-action`. If no `asp-action` is specified then the default `asp-controller` will be the current executing controller.
+`asp-action` is the name of the action method in the controller that will be included in the generated `href`. For example, the following code set the generated `href` to point to the speaker detail page:
+
+```html
+<a asp-controller="Speaker" asp-action="Detail" >Speaker Detail</a>
+```
+
+The generated URL will be
```

Comment:
@Rick-Anderson Passing quick ?: Are the style manual guidelines on present/past tense somewhat relaxed for the two docs repos? You know, they warn off of *will*, *was*, and verbs ending in *-ed*. However, it's not very uncommon to see the tenses change when writing in a conversational tone and telling the reader that 'this will happen', especially in tutorial/walk-through topics. I noticed that reviews don't necessarily call tense out here and over on .NET Core docs, so just wondering if that was a relaxed provision?